content/docs/languages/go/basics.md

```diff
@@ -558,22 +561,49 @@ any order — the streams operate completely independently.
 
 ### Try it out!
 
-Work from the example directory:
+Execute the following commands from the `examples/route_guide` directory:
 
-```sh
-$ cd $GOPATH/src/google.golang.org/grpc/examples/route_guide
-```
+ 1. Run the server:
 
-Run the server:
+    ```sh
+    $ go run server/server.go
+    ```
 
-```sh
-$ go run server/server.go
-```
+ 2. From another terminal, run the client:
 
-From a different terminal, run the client:
+    ```sh
+    $ go run client/client.go
+    ```
 
-```sh
-$ go run client/client.go
+You'll see output like this:
+
+```nocode
+Getting feature for point (409146138, -746188906)
+name:"Berkshire Valley Management Area Trail, Jefferson, NJ, USA" location:<latitude:409146138 longitude:-746188906 >
+Getting feature for point (0, 0)
+location:<>
+Looking for features within lo:<latitude:400000000 longitude:-750000000 > hi:<latitude:420000000 longitude:-730000000 >
+name:"Patriots Path, Mendham, NJ 07945, USA" location:<latitude:407838351 longitude:-746143763 >
+...
+name:"3 Hasta Way, Newton, NJ 07860, USA" location:<latitude:410248224 longitude:-747127767 >
+Traversing 56 points.
+Route summary: point_count:56 distance:497013163
+Got message First message at point(0, 1)
+Got message Second message at point(0, 2)
+Got message Third message at point(0, 3)
+Got message First message at point(0, 1)
+Got message Fourth message at point(0, 1)
+Got message Second message at point(0, 2)
+Got message Fifth message at point(0, 2)
+Got message Third message at point(0, 3)
+Got message Sixth message at point(0, 3)
 ```
 
+{{< note >}}
+  We've omitted timestamps from the client and server trace output shown in this
+  page.
+{{< /note >}}
+
+[download]: https://github.com/grpc/grpc-go/archive/master.zip
+[grpc-go]: https://github.com/grpc/grpc-go
 [Quick start]: ../quickstart/
```

Comment:
Which URL would you like to see instead? Actually, ... what do you mean? So that one can navigate the raw `.md` pages from GitHub? We don't support that for Hugo content pages -- folks should be navigating the site, not the repo source.

Reply:
I'm wondering if we can find another way of linking, or restructure things to make the site compatible with github so that both can use the same links.  Not to make the content browsable that way, but so that reviewers can confirm the links will work before we push it live to the site.  If that's not possible, or just way way out of scope, that's fine.

Reply:
> ... so that reviewers can confirm the links will work

As you might know, every PR gets deployed: for this PR the URL of interest is https://deploy-preview-450--grpc-io.netlify.app/docs/languages/go/basics/. Links can be checked in that way.

We haven't setup automated link checking for this site, but I routinely run a link checker on the entire site. Another option is mentioned in #441.

> ... make the site compatible with github so that both can use the same links

Yes, I'd say that that is out of scope.

Reply:
> As you might know, every PR gets deployed 

I had forgotten about that, thanks.